docs/topics/pages.md

```diff
@@ -335,18 +337,19 @@ Wagtail can nest the content of other models within the page. This is useful for
 
 Each inline model requires the following:
 
-- It must inherit from {class}`wagtail.models.Orderable`
-- It must have a `ParentalKey` to the parent model
+-   It must inherit from {class}`wagtail.models.Orderable`
+-   It must have a `ParentalKey` to the parent model
 
-```{note} django-modelcluster and ParentalKey
+````{note} django-modelcluster and ParentalKey
 The model inlining feature is provided by [django-modelcluster](https://github.com/torchbox/django-modelcluster) and the `ParentalKey` field type must be imported from there:
 
 ```python
     from modelcluster.fields import ParentalKey
-```
+````
```

Comment:
this doc seems to get extra backticks for this code block and below the `ParentalKey` line

Reply:
Hmm ok I'll have to run this locally to check. Probably an issue with the code snippet within the admonition.

Reply:
should be fixed now, the actual rendering was working better than the current production docs but I have fixed up the odd extra backtick.

Note: We need four backticks on the outer admonition due to there being three for the inner code snippet.

Here is the validation screenshot (with the newly pushed changes), left is current stable, right is local docs.
![Screen Shot 2022-06-01 at 8 40 52 pm](https://user-images.githubusercontent.com/1396140/171386508-c12e0d4c-5e79-429d-bc5d-464e65486627.png)
See that the text 'For example, the following inline model can be used to add related links (a list of name, url pairs) to the `BlogPage` model:' is incorrectly showing inside the code block in the current stable docs.